enterprise/server/test/integration/remote_execution/remote_execution_test.go

```diff
@@ -1860,7 +1873,7 @@ fi`, fname)
 	require.Equal(t, op1, op3, "expected action to be merged")
 
 	// Let the hedged execution finish.
-	time.Sleep(10 * time.Millisecond)
+	time.Sleep(100 * time.Millisecond)
```

Comment:
No need to do this right now (since this PR seems like an improvement over what we've got today) but instead of using shell scripts and coordinating this logic using sleeps (which are likely to fail under load), we could use `RunInterceptor` to write the execution logic in Go and do this coordination e.g. using channels.